uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Hook/PageRun/CustomFieldIds.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class CRM_Tasksassignments_Hook_PageRun_CustomFieldIds {
+
+  private $resources;
+
+  /**
+   * @param CRM_Core_Resources $resources
+   */
+  public function __construct(CRM_Core_Resources $resources) {
+    $this->resources = $resources;
+  }
+
+  /**
+   * Provides the IDs for custom fields needed by specific Task and Workflow pages.
+   *
+   * @param CRM_Core_Page $page
+   */
+  public function handle($page) {
+    if (!$this->shouldHandle($page)) {
+      return;
+    }
+
+    $this->resources->addVars('customFieldIds', [
+      'caseType.category' => CRM_Core_BAO_CustomField::getCustomFieldID('category', 'case_type_category'),
```

Comment:
Seeing as this line is a little long could you move the `CRM_Core_BAO_CustomField::getCustomFieldID('category', 'case_type_category'),` to a variable on a different line?